ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -843,6 +875,24 @@ public class BottomSheetController: UIViewController {
         return maxHeight
     }
 
+    // Output of `collapsedHeightResolver` wrapped in a cache.
+    private var resolvedCollapsedSheetHeight: CGFloat {
+        let oldContext = lastCollapsedSheetHeightResolutionContext
```

Comment:
Probably could do with a 

```swift
guard let oldContext = lastCollapsedSheetHeightResolutionContext else {
    return lastResolvedCollapsedSheetHeight
}
```

Then the logic below would simplify a fair bit.

Reply:
That seems like it's different logic? If there's no previously used context, then last resolved height will have no meaning, so we need to go to the resolver to get a fresh value.